velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -713,6 +759,32 @@ MapVectorPtr VectorFuzzer::fuzzMap(
       values);
 }
 
+FlatMapVectorPtr VectorFuzzer::fuzzFlatMap(
+    const TypePtr& keyType,
+    const TypePtr& valueType,
+    vector_size_t size) {
+  // All keys present in the entirety of the FlatMap column.
+  VectorPtr distinctKeys =
+      fuzzUniqueFlatNotNull(keyType, getElementsVectorLength(opts_, size));
```

Comment:
shouldn't this be `opts.containerLength` or `opts.complexElementsMaxSize`? This has to do with the number of key value pairs per record, not overall (times `size`).